eagerpy/tensor/pytorch.py

```diff
@@ -203,6 +203,12 @@ def argsort(self: TensorType, axis: int = -1) -> TensorType:
     def sort(self: TensorType, axis: int = -1) -> TensorType:
         return type(self)(self.raw.sort(dim=axis).values)  # type: ignore
 
+    def topk(
+        self: TensorType, k: int, sorted: bool = True
+    ) -> Tuple[TensorType, TensorType]:
+        pair = self.raw.topk(k, sorted=sorted)
```

Comment:
You can use tuple unpacking instead: `values, indices = self.raw…`.
It's more descriptive and avoids tuple indexing in the next line.